WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -227,6 +227,7 @@ extension PointOfSaleAggregateModel {
         }
         do {
             try await collectPayment(for: order)
+            collectOrderPaymentAnalyticsTracker.resetCheckoutTapCountTracker()
```

Comment:
Could tracker reset all properties when `trackSuccessfulPayment` is called?

Reply:
Good point, updated here: https://github.com/woocommerce/woocommerce-ios/pull/15148/commits/b0465370292409b0dea466bfcca0043e59496e87 . Will also move the cash bit in the next PR, as there's a "successful cash payment" function too